cli/src/main/java/org/aya/cli/repl/RenderOptions.java

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2020-2022 Tesla (Yinsen) Zhang.
+// Use of this source code is governed by the MIT license that can be found in the LICENSE.md file.
+package org.aya.cli.repl;
+
+import com.google.gson.*;
+import kala.collection.immutable.ImmutableSeq;
+import kala.collection.mutable.MutableMap;
+import kala.control.Result;
+import org.aya.pretty.printer.ColorScheme;
+import org.aya.pretty.printer.StyleFamily;
+import org.aya.pretty.style.AyaColorScheme;
+import org.aya.util.reporter.Problem;
+import org.aya.util.reporter.Reporter;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.lang.reflect.Type;
+import java.util.Map;
+
+public record RenderOptions(
+  @NotNull ColorScheme colorScheme,
+  @NotNull StyleFamily styleFamily
+) {
+  public final static class Deserializer implements JsonDeserializer<RenderOptions>, JsonSerializer<RenderOptions> {
+    public static class ColorSchemeJson {
+      public @Nullable java.util.Map<String, String> definedColors;
```

Comment:
Why full qualified name

Reply:
Because I used both kala and java.util

Reply:
Might be a reason to migrate to jackson...

Reply:
> Why `java.util`? Is it because it's used in Gson?

Yes.